sass/_themes.scss

```diff
@@ -68,13 +69,14 @@ $color-red-4: #d33636;
 $color-red-5: #eb3e48;
 $color-red-6: #ff444f14;
 $color-red-7: #ff525c;
-$color-grey: #c2c2c2;
+$COLOR_LIGHT_GREY: rgba(0, 0, 0, 0.08);
```

Comment:
Can we maybe move this color to the top section and rename it? I see color_light_gray there, how come we need both of them?

Reply:
Fixed